client/src/components/PaginationController.tsx

```diff
@@ -0,0 +1,87 @@
+import { MouseEventHandler } from 'react';
+import styled from 'styled-components';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { icon } from '@fortawesome/fontawesome-svg-core/import.macro';
+
+interface PaginationControllerProps {
+  paginationRange: number[];
+  currentPage: number;
+  setCurrentPage: Function;
+}
+
+const PaginationController = ({
+  paginationRange,
+  currentPage,
+  setCurrentPage,
+}: PaginationControllerProps) => {
+  // If there are less than 2 times in pagination range we shall not render the component
```

Comment:
can you reword to 
"Do not render component if only one page"
or something a bit simpler to understand?